mne/time_frequency/tfr.py

```diff
@@ -505,7 +505,8 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     from ..viz.utils import _setup_vmin_vmax
 
     copy = baseline is not None
-    data = rescale(data, times, baseline, mode, copy=copy)
+    if mode is not None:
+        data = rescale(data, times, baseline, mode, copy=copy)
```

Comment:
There is already a check for "if baseline is None do nothing" so by leaving
this check here you're unnecessarily splitting the "do nothing" checks, and
potentially forcing similar code to go elsewhere in the future